src/stories/components/custom-table/list-core-unit/list-core-unit.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { columns, CoreUnit } from '@ses/core/utils/test.utils';
+
+import ListCoreUnit from './list-core-unit';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/CUTable/ListCoreUnit',
+  component: ListCoreUnit,
+  parameters: {
+    chromatic: {
+      viewports: [1194, 1280, 1440],
```

Comment:
I don't see any difference in the component UI in the breakpoints 1194 and 1280, so there should be one of them only